skillsmap/src/components/CompletionModal.tsx

```diff
@@ -0,0 +1,44 @@
+/// <reference path="../lib/skillMap.d.ts" />
+
+import * as React from "react";
+import { connect } from 'react-redux';
+
+import { SkillsMapState } from '../store/reducer';
+import { dispatchHideCompletionModal } from '../actions/dispatch';
+
+import { Modal } from './Modal';
+
+interface CompletionModalProps {
+    activity?: MapActivity;
+    dispatchHideCompletionModal: () => void;
+}
+
+export class CompletionModalImpl extends React.Component<CompletionModalProps> {
+    render() {
+        const  { activity, dispatchHideCompletionModal } = this.props;
+        if (!activity) return <div />
+
+        const actions = [
+            { label: "NEXT", onClick: () => console.log(activity?.next?.[0]?.activityId || "no next") },
+            { label: "REWARDS", onClick: () => console.log("rewards") }
+        ]
+
+        return <Modal title="Activity Complete!" actions={actions} onClose={() => dispatchHideCompletionModal()}>
```

Comment:
Maybe we should pull these strings out into variables or expressions so that localization is easier later